src/booleans.js

```diff
@@ -1,62 +1,94 @@
 function negate(a) {
   // your code here
+  return !(a > 0);
 };
 
 function both(a, b) {
   // your code here
+  return a > 0 && b > 0;
```

Comment:
As above, you shouldnt try to compare booleans with numerical values like this. Since you comparing 2 booleans it could be written:
```
  return a && b;
```